backend/app/routers/experiments.py

```diff
@@ -47,27 +45,32 @@ async def get_experiments(
     response_model=int,
 )
 async def get_experiments_count(
-    user: Json = Depends(get_current_user_optional),
-    include_mine: bool = False,
-    query_operator: QueryOperator = QueryOperator.AND,
+    user: Annotated[dict, Depends(get_current_user(required=False))],
 ) -> Any:
-    result_set = find_specific_experiments(
-        include_mine=include_mine, query_operator=query_operator, user=user
-    )
-    return await result_set.count()
+    # TODO hotfix for returning only my experiments
+    if user:
+        return await Experiment.find(Experiment.created_by == user["email"]).count()
```

Comment:
It would be good to either have a method like "user_experiments" or to have a test. Maybe in the next run?

Reply:
You mean something like `Experiment.find_by_user(user)`? I would like that, but we would also need to create a proper User class as it is only dict for now. If you agree I can add it in a separate PR.